### ISSUE TITLE
Missing -race flag

### ISSUE THREAD

**Issue by lancecarlson** [constabulary/gb#543]
I'm assuming this because I have an outdated binary somehow. When I run gb help build, the flag doesn't appear:

```sh
$ gb help build
usage: gb build [build flags] [packages]

Build compiles the packages named by the import paths, along with their
dependencies.

Flags:

	-f
		ignore cached packages if present, new packages built will overwrite
		any cached packages. This effectively disables incremental
		compilation.
	-F
		do not cache packages, cached packages will still be used for
		incremental compilation. -f -F is advised to disable the package
		caching system.
	-q
		decreases verbosity, effectively raising the output level to ERROR.
		In a successful build, no output will be displayed.
	-P
		The number of build jobs to run in parallel, including test execution.
		By default this is the number of CPUs visible to gb.
	-R
		sets the base of the project root search path from the current working
		directory to the value supplied. Effectively gb changes working
		directory to this path before searching for the project root.
	-v
		increases verbosity, effectively lowering the output level from INFO
		to DEBUG.
	-dotfile
		if provided, gb will output a dot formatted file of the build steps to
		be performed.
	-ldflags 'flag list'
		arguments to pass on each linker invocation.
	-gcflags 'arg list'
		arguments to pass on each go tool compile invocation.
```

I tried upgrading using go get github.com/constabulary/gb/... but that didn't see to help.

**Comment by davecheney** [comment]
I could have forgotten to add it to the help text. If so, i'll add that
documentation and send a PR to close this issue.

**Comment by lancecarlson** [comment]
Is this where it's supposed to be?

https://github.com/constabulary/gb/blob/master/cmd/gb/help.go

**Comment by davecheney** [comment]
It looks like this was fixed in 56a6bd0b
